source/partials/release_notes/_release-17-7-0.md.erb

```diff
@@ -0,0 +1,4 @@
+<h4>Security Fixes</h4>
+
+**Important:** This release comes on the back of 17.6.0 as we have identified an extremely critical security vulnerability with 17.5.0 and 17.6.0. If you are on 17.5.0 or 17.6.0, immediately upgrade to this version to keep your GoCD server secure. We want to give sometime for users to upgrade before disclosing the details of the vulnerability.
```

Comment:
How about this: "If you are on 17.5.0 or 17.6.0, please upgrade immediately to this version"...